script/Arcueid/Day 13/13_00_ARC13_4.txt

```diff
@@ -1359,12 +1357,12 @@ Arcueid doesn't say anything.
 [sha:a6f71711381251cbfb8a786a5d559d3d525b2a12]{
 -- Page 386, Offset 32752.
 -- ただ、見落としてしまいそうなほど<微|かす>かに。
-Her response is so feeble as to be almost imperceptible―――
+Instead, she responds so feebly that it's almost imperceptible―――
 }
 [sha:d7519b998ff1580d958b88b87b816bedcc1355c8]{
 -- Page 386, Offset 32753.
 -- 首を、横にふった。
-But she is shaking her head.
+By shaking her head.
```

Comment:
On the fence about this one but will sleep on it

Reply:
it's fine i guess, reads a little awkwardly